cli/cmd/get.go

```diff
@@ -244,20 +244,19 @@ func describeAPI(name string, resourcesRes *schema.GetResourcesResponse, flagVer
 		Rows:    [][]interface{}{row},
 	}
 
-	var predictionMetrics string
+	var out string
 	apiMetrics, err := getAPIMetrics(ctx.App.Name, api.Name)
-	if err != nil || apiMetrics == nil {
-		predictionMetrics = "\nmetrics are not available yet\n"
-	} else {
-		statusTable = appendNetworkMetrics(statusTable, apiMetrics)
-		if api.Tracker != nil {
-			predictionMetrics = "\n" + predictionMetricsTable(apiMetrics, api) + "\n"
+	statusTable = appendNetworkMetrics(statusTable, apiMetrics) // adds blank stats when there is an error
+	out = "\n" + table.MustFormat(statusTable) + "\n"
+
+	if err != nil {
+		if !strings.Contains(err.Error(), "api is still initializing") {
```

Comment:
Is it possible to use cast the error and check `ErrKind == ErrAPIInitializing`? There are a few ways we could do this, let's discuss